content/docs/setup/kubernetes/additional-setup/mesh-expansion/gateways/index.md

```diff
@@ -0,0 +1,459 @@
+| service                            | istiocoredns                     | Resolve *.global to Istio Ingress gateway    |
+| gateway.networking.istio.io        | meshexpansion-gateway            | Open port for Pilot, Citadel and Mixer       |
+| gateway.networking.istio.io        | istio-multicluster-egressgateway | Open port 15443 for outbound *.global traffic|
+| gateway.networking.istio.io        | istio-multicluster-ingressgateway| Open port 15443 for inbound *.global traffic |
+| envoyfilter.networking.istio.io    | istio-multicluster-ingressgateway| Transform *.global to *. svc.cluster.local   |
+| destinationrule.networking.istio.io| stio-multicluster-destinationrule| Set traffic policy for 15443 traffic         |
+| destinationrule.networking.istio.io| meshexpansion-dr-pilot           | Set traffic policy for `istio-pilot`         |
+| destinationrule.networking.istio.io| istio-policy                     | Set traffic policy for `istio-policy`        |
+| destinationrule.networking.istio.io| istio-telemetry                  | Set traffic policy for `istio-telemetry`     |
+| virtualservice.networking.istio.io | meshexpansion-vs-pilot           | Set route info for `istio-pilot`             |
+| virtualservice.networking.istio.io | meshexpansion-vs-citadel         | Set route info for `istio-citadel`           |
+
+## Expose service running on cluster to VMs
+
+Every service in the cluster that needs to be accessed from the VM requires a ServiceEntry configuration in the cluster. The host used in the service entry should be of the form <name>.<namespace>.global where name and namespace correspond to the service’s name and namespace respectively.
+
+To demonstrate access from VM to  cluster services, configure the
+the [httpbin service]({{<github_tree>}}/samples/httpbin)
+in the cluster.
+
+1. Deploy the `httpbin` service in the cluster
+
+    {{< text bash >}}
+    $ kubectl create namespace bar
+    $ kubectl label namespace bar istio-injection=enabled
+    $ kubectl apply -n bar -f @samples/httpbin/httpbin.yaml@
+    {{< /text >}}
+
+1. Create a service entry for the `httpbin` service in the cluster.
+
+    To allow services running in VM  to access `httpbin` in the cluster, we need to create
+    a service entry for it. The host name of the service entry should be of the form
+    `<name>.<namespace>.global` where name and namespace correspond to the
+    remote service's name and namespace respectively.
+
+    For DNS resolution for services under the `*.global` domain, you need to assign these
+    services an IP address.
+
+    {{< tip >}}
+    Each service (in the `.global` DNS domain) must have a unique IP within the cluster.
+    {{< /tip >}}
+
+    If the global services have actual VIPs, you can use those, but otherwise we suggest
+    using IPs from the loopback range `127.0.0.0/8` that are not already allocated.
```

Comment:
`from the node-local address pool` instead of `from the loopback range`

`that are not already allocated` `that are unique in the network`.